### PR TITLE
Update post audit

### PR DESCRIPTION
**Pull request by alexvandesande**
Mmm I feel there are some diffs there that I didn't expect. I'll take a closer look

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -278,6 +284,37 @@ Registrar.prototype.getEntry = function getEntry(input, callback) {
...
+  registrarContract.getAllowedTime(hash, (timeErr, timeResult) => {
+    if (callback) {
+      callback(null, timeResult);
+    } else {
+      return timeResult;
+    }
+  });
+};
+
+/**
```
Comment (Arachnid) [inline src/index.js:283]: ?

**Comment by maurelian** [comment]
> Mmm I feel there are some diffs there that I didn't expect. I'll take a closer look

OK. LMK.

File: .meteor-portable [hunk 2]
```diff
@@ -0,0 +1 @@
+false
```
Comment (maurelian) [inline .meteor-portable]: What does this do?

File: src/index.js [hunk 3]
```diff
@@ -1,9 +1,11 @@
-const interfaces = require('./interfaces.js');
+'use strict';
 
-const ENS = require('ethereum-ens');
+var interfaces = require('./interfaces.js');
```
Comment (maurelian) [inline src/index.js]: Based on the changes in this file, it almost looks like it's been processed by babel to ES5?

File: src/index.js (hunk 1 above)
Comment (maurelian) [reply src/index.js:283]: What does the question mark mean @Arachnid?

File: src/test/dotEthRegistrar.sol [hunk 4]
```diff
@@ -1,603 +0,0 @@
-pragma solidity ^0.4.0;
```
Comment (maurelian) [inline src/test/dotEthRegistrar.sol]: This file was used for testing.

**Review by maurelian — COMMENTED** [review]
@alexvandesande The tests no longer run here, and all the es6 features have been taken out.  It seems like this is in the name of meteor support, but the build process should provide a `/lib/index.js` which supports meteor, if not, we could add a `build:meteor` script to the package.json file. 

Generally, I'm curious what the motivation is for merging this in? I'd like this library to be general purpose, and useful to anyone else building a dapp to interact with the registrar. If these changes are necessary, maybe it would make sense to just allow these repos to fork?

**Comment by alexvandesande** [comment]
@maurelian @Arachnid I've removed all of the code I didn't recognize that I was adding back by accident. 

I tried to update the tests, but they still fail at "normalize is not a function"

I bumped the version to 0.8.0 because it breaks some compatibility

**Comment by maurelian** [comment]
> I tried to update the tests, but they still fail at "normalize is not a function"

Odd. I don't get that error. At what point in the testing do you see that? 

I'm having a different issue actually, which is that I can't get the `getEntry()` function to return anything other than `entry.status = 5` (not-yet-available).

I even fast-forwarded the testrpc EVM by 8 weeks, and still names are all coming back as not-yet-available. I'm super confused by it.

**Comment by alexvandesande** [comment]
That's a weird issue. I've noticed that if you forward time on testrpc but do not submit a transaction, it will not effectively forward it. Try forwarding 8 weeks and then starting an auction to see if it works

**Comment by maurelian** [comment]
OK, thanks that was helpful. I had to update testrpc.

I added a bunch of fixes to the test suite onto this, pretty sure nothing changed in the actual lib.

**Pull request merged**